src/components/ProgressRow/ProgressRow.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import './ProgressRow.css';
+import Board from "../Board/Board";
+import useTaskStatuses from "../../hooks/useTaskStatuses";
+
+const ProgressRow = ({ tasks, setTasks }) => {
+    const { todo, readyForDev, inProgress, done } = useTaskStatuses(tasks);
```

Comment:
maybe call hook inside the <Board/> so we can refuse  having so much props?

Reply:
done